panels/datetime/cc-datetime-panel.c

```diff
@@ -359,8 +369,12 @@ update_time (CcDateTimePanel *self)
   /* Update the time on the listbow row */
   if (use_ampm)
     {
+      gchar *format_string = g_strdup_printf ("%s %s", _("%e %B %Y, %l:%M"),
```

Comment:
In general these changes look good, but you should try to preserve all the existing translatable strings, instead of changing them.
You can just use the `fallback_ampm` variable and add a new string when it's true, using the old one when it's not.